app/lnd/lib/util.js

```diff
@@ -10,6 +11,45 @@ import { mainLog } from '../../utils/log'
 const fsReadFile = promisify(fs.readFile)
 const dnsLookup = promisify(dns.lookup)
 
+/**
+ * Helper function to get the current block height.
+ * @return {Number} The current block height.
+ */
+export const fetchBlockHeight = () => {
+  const sources = [
+    {
+      baseUrl: `https://testnet-api.smartbit.com.au/v1/blockchain/blocks?limit=1`,
+      path: 'blocks[0].height'
+    },
+    {
+      baseUrl: `https://tchain.api.btc.com/v3/block/latest`,
+      path: 'data.height'
+    },
+    {
+      baseUrl: `https://api.blockcypher.com/v1/btc/test3`,
+      path: 'height'
+    }
+  ]
+  const fetchData = (baseUrl, path) => {
+    mainLog.info(`Fetching current block height from: ${baseUrl}`)
+    return axios({
+      method: 'get',
+      timeout: 5000,
+      url: baseUrl
+    })
+      .then(response => {
+        const height = Number(path.split('.').reduce((a, b) => a[b], response.data))
+        mainLog.info(`Fetched block height as ${height} from: ${response.config.url}`)
+        return height
+      })
+      .catch(() => null)
+  }
+
+  const promises = []
+  sources.forEach(source => promises.push(fetchData(source.baseUrl, source.path)))
```

Comment:
nit: `map`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
lint requires it

Reply:
updated